### ISSUE TITLE
Pick a Front End technology

### ISSUE THREAD

**Issue by thirtyaughtsix** [computablelabs/ffa-frontend#2]
Front end shootout: React, Vue, Angular.

**Comment by thirtyaughtsix** [comment]
Caveat: It's extraordinarily unlikely that FFA will ever attain a complexity where the choice of front end framework will itself separate success from failure, so this is somewhat academic...

That said, and first off, Angular still `ng-blows`.  I `ng-can't` `ng-stand-it`.  `ngrx`remains impressive but so what?  The JVM is also a marvel of engineering and coding Java is like peeling off and eating your own skin (and that's before you enter the fifth circle of maven hell).

Angular fans still sell the framework as _The ***Only*** Enterprise Solution™_.  After 20+ years in the "enterprise" trenches, I don't even understand what that's supposed to mean.  I think it means: garbage-ware for group-think developers whose life is a Sisyphean tragicomedy of beating back corporate metastasis with technology.  Such a show might get greenlighted on the Fox network but it's antithetical to our _weltanschauung_. 

So then, React.  Or rather: why not React?  I think the primary demur is that it has become unacceptably bloated, a hog as it were.  And as was once said of the late Jimmy Hoffa: _pigs get fed, hogs get slaughtered._  

It is an axiom of our industry that by the time a technology becomes _The Default Choice℠_, other, younger options lie in wait to usurp leadership of the Teamsters.  Vue leads the pack of such alternatives.  It works more or less like React, supports Typescript "natively," and has very nice tooling.  Vue would close the door on esoteric stuff like Purescript and ReasonML but it leaves open the possibility of WASM (although WASM isn't specific to Vue as there are React bindings/integrations for WASM as well).  While I do love me some functional, I accept this trade.  We can always enforce a more functional-like Typescript syntax by bolting on Ramda/Rambda/Immutable and holding merciless code reviews that would delight de Torquemada.

tl;dr - The Tao of Computable says _Never build anything more than what you need_ and thus Vue fits the bill better than either React or Angular.

**Comment by thirtyaughtsix** [comment]
Team has agreed on vue.
